src/evtx_chunk.rs

```diff
@@ -179,23 +185,12 @@ impl<'a> Iterator for IterChunkRecords<'a> {
 
         trace!("Need to deserialize {} bytes of binxml", binxml_data_size);
         let deserializer = BinXmlDeserializer::init(
-            &self.chunk.data,
+            self.chunk.data,
             self.offset_from_chunk_start + cursor.position(),
-            &self.chunk.string_cache,
+            self.chunk.string_cache,
             &self.chunk.template_table,
```

Comment:
document why `&self.chunk.template_table` is passed by reference, vs `self.chunk.string_cache` which is moved (in this case it's a reference to string cache, maybe we should consider a better name).